bindings/python/ckzg.c

```diff
@@ -129,16 +130,19 @@ static PyObject* verify_kzg_proof_wrap(PyObject *self, PyObject *args) {
   const Bytes32 *y_bytes = (Bytes32 *)PyBytes_AsString(y);
   const Bytes48 *proof_bytes = (Bytes48 *)PyBytes_AsString(p);
 
-  bool ok;
-  if (verify_kzg_proof(&ok,
-        commitment_bytes, z_bytes, y_bytes, proof_bytes,
-        PyCapsule_GetPointer(s, "KZGSettings")) != C_KZG_OK) {
-    return PyErr_Format(PyExc_RuntimeError, "verify_kzg_proof failed");
+  C_KZG_RET ret = verify_kzg_proof(
+                                   commitment_bytes, z_bytes, y_bytes, proof_bytes,
+                                   PyCapsule_GetPointer(s, "KZGSettings"));
```

Comment:
This formatting looks a bit strange. Is this intentional?

Reply:
Fixed in 1400821 !